.gitattributes

```diff
@@ -1,3 +1,4 @@
+/bin/ export-ignore
```

Comment:
There's not a trailing slash on `tests/` below, which is more correct?

Reply:
the trailing slash enforce matching only a directory. Without it, it could also match a file named `bin` at the root.

As we cannot have both a folder and a file with the same name at the root anyway (and that we match only at the root due to the leading slash), I would say that both are equally correct.